src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -526,8 +525,8 @@ class NetworkOPsImp final
     std::atomic <bool> mNeedNetworkLedger;
     bool m_amendmentBlocked;
 
-    DeadlineTimer m_heartbeatTimer;
-    DeadlineTimer m_clusterTimer;
+    boost::optional<boost::asio::steady_timer> heartbeatTimer_;
+    boost::optional<boost::asio::steady_timer> clusterTimer_;
```

Comment:
The `optional` is a cheaper version of `unique_ptr` that doesn't allocate.  We need to postpone the construction of the timers until after the `NetworkOPs` constructor runs because we shouldn't call `app_` until `NetworkOPs` is fully constructed.  At least that's the intention.